internal/tiltfile/tiltfile_test.go

```diff
@@ -3239,42 +3239,44 @@ func TestDockerbuildIgnoreAsString(t *testing.T) {
 	f := newFixture(t)
 	defer f.TearDown()
 
-	f.dockerfile("foo/Dockerfile")
-	f.yaml("foo.yaml", deployment("foo", image("fooimage")))
-
+	f.dockerfile("Dockerfile")
+	f.yaml("foo.yaml", deployment("foo", image("gcr.io/foo")))
 	f.file("Tiltfile", `
 k8s_resource_assembly_version(2)
-docker_build('fooimage', 'foo', ignore="package.json")
+docker_build('gcr.io/foo', '.', ignore="*.txt")
 k8s_yaml('foo.yaml')
 `)
 
-	f.load()
-
+	f.load("foo")
```

Comment:
hmm any particular reason why you're only loading "foo"? (i.e. why you're not leaving this call as `f.load()`?)

Reply:
Nope no real reason, just copy pasta :D